processing/data_collection/gazette/spiders/base.py

```diff
@@ -1,7 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we can remove these lines. We use python3 and this is not required anymore because the default is utf-8.

Reply:
will fix this and the merge conflict